pkgs/by-name/py/pyditz/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pyditz";
+  version = "0.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-2gNlrpBk4wxKJ1JvsNeoAv2lyGUc2mmQ0Xvn7eiaJVE=";
+  };
+
+  nativeBuildInputs = with python3Packages; [ setuptools-scm ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pyyaml
+    six
+    jinja2
+    cerberus
+  ];
+
+  meta = {
+    homepage = "https://pypi.org/project/pyditz/";
+    description = "Drop-in replacement for the Ditz distributed issue tracker";
+    maintainers = with lib.maintainers; [ ilikeavocadoes ];
+    license = lib.licenses.lgpl2;
```

See https://hg.sr.ht/~zondo/pyditz/browse/setup.cfg?rev=tip#L18

```suggestion
    license = lib.licenses.lgpl2Plus;
```